src/models/fields/IllnessStartDate.schema.ts

```diff
@@ -0,0 +1,50 @@
+import * as Joi from '@hapi/joi';
+
+const illnessStartDayErrorMessage: string = 'You must enter a day';
+const illnessStartMonthErrorMessage: string = 'You must enter a month';
+const illnessStartYearErrorMessage: string = 'You must enter a year';
+const invalidDate: string = 'Enter a real date';
+const invalidDateFuture: string = 'Start date must be today or in the past';
+
+const dayMonthRegex: RegExp = /^[0-9]{1,2}$/i;
```

Comment:
`i` modifier making regex case insensitive does not make much sense here since only numbers are allowed, does it?